lib/tasks/check_sessions.rake

```diff
@@ -0,0 +1,41 @@
+#
+# usage:
+#   % rake check_stale_sessions [DEBUG=1] [PURGE=1]
+#
+desc 'check for stale sessions'
+task check_stale_sessions: :environment do
+  puts "now:               #{Time.zone.now}" if debug
+  puts "stale_window_time: #{stale_window_time}" if debug
+  ActiveRecord::SessionStore::Session.where('updated_at < ?', stale_window_time).each do |session|
+    next unless session.data.keys.any?
+    session.data.symbolize_keys!
+    puts "pinged_at:         #{session.data[:pinged_at]}" if debug
+    puts session.data.pretty_inspect if debug
+    if stale?(session.data)
+      puts " ----------> stale!" if debug
+      session.destroy! if purge
+    end
+  end
+end
+
+def purge
+  ENV['PURGE'].present? && ENV['PURGE'] == 1.to_s
```

Comment:
The presence check is not necessary here. `ENV['PURGE'] == '1'` will return false if `PURGE` is not set.